### ISSUE TITLE
Add Visual Works like Inspector

### ISSUE THREAD

**Issue by brunobuzzi** [jgfoster/Jade.bak#69]
In Visual Works a Inspector open only once. Further inspections are in the same window. But you have navigation buttons to go backward and forward. 
In Jade it would be like add "recently visited objects" to the Inspector. 
Which is the best behavior:
* A User Preference window where the user select what inspector use (Normal/Navigation Inspector)
* Put the option in both Inspector to become the other.
I think the lastone maybe more practical.
![imagen](https://cloud.githubusercontent.com/assets/4719282/23574392/a900d806-005c-11e7-8e5b-8d3c479a160c.png)

**Comment by brunobuzzi** [comment]
Almost done. We have to implement how to configure the Inspector we want to use (Normal/Navigation Inspector)
![imagen](https://cloud.githubusercontent.com/assets/4719282/23583055/d8f3528a-0117-11e7-9ec0-d2636c2ad10f.png)

**Comment by jgfoster** [comment]
I think this is done. If we discover other issues we can reopen this or open a new issue.
